orangecontrib/spectroscopy/widgets/peak_editors.py

```diff
@@ -354,6 +317,72 @@ def model_lines():
         """
         raise NotImplementedError
 
+    @classmethod
+    def _translate(cls, name, hints):
+        hints = hints.copy()
+        defaults = cls.defaults()[name]
+
+        vary = hints["vary"]
+        delta = hints.get("delta", DEFAULT_DELTA)
+        value = hints.get("value", DEFAULT_VALUE)
+
+        # special delta case
+        if vary == 'delta' and delta == 0:
+            vary = 'fixed'
+
+        if vary == 'delta':
+            hints["min"] = value - delta
+            hints["max"] = value + delta
+        elif vary == 'limits':
+            hints["value"] = value
```

Comment:
I can't see the point of this line, but it seems harmless

Reply:
Now the Editor only saves things to dict that are specified. If they are left as defaults, they do not get into dict. But UI still shows you something, specifically, the default 0. This like is there to force that this 0 from the UI actually gets into lmfit, otherwise we would be relying that lmfit's default is also 0. Well, if that is so (I did not check) this line could be removed. :) I rather went for something more explicit here. Good catch, btw!

Reply:
Often the `lmfit` defaults we care about are actually `1` rather than `0` (i.e. Gaussian sigma['value']). But I checked and at least in the GUI they are still being set correctly somewhere else. As long as that default is propagating to the `lmfit` Model creation (and it seems it is) we should be good.